transportation-on-vacation.js

```diff
@@ -0,0 +1,7 @@
+function rentalCarCost(d) {
+    if(d>6)
```

Comment:
This works, but you should always take the code indentation and separation in account

`return (d*40)-50;`